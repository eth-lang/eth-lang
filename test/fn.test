(fn () ())
(fn (a) ())
(fn (a b c d e) ())
(fn (a ... as) ())
(fn (... as) ())
