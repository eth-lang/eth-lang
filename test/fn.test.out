(function () {
  return null;
  });
(function (a) {
  return null;
  });
(function (a, b, c, d, e) {
  return null;
  });
(function (a) {
  var as = Array.prototype.slice.call(arguments, 1);
  return null;
  });
(function () {
  var as = Array.prototype.slice.call(arguments, 0);
  return null;
  });

